app/assets/javascripts/components/form/form_body.es6.jsx

```diff
@@ -8,8 +8,15 @@ class FormBody extends Component {
     return {};
   }
 
+//TODO(Sonia): Add more styling to container
   get styles() {
-    return {};
+    return {
+      container: Object.assign( 
+        {},
+        StyleConstants.cards.default,
+        {},
```

Comment:
@sonia-y does this trailing comma create an error? Totally my bad I forgot that this was in a `Object.assign` call... If it doesn't break anything though that'd be an interesting revelation.